todo_app/app.py

```diff
@@ -3,40 +3,221 @@
 
 
 from todo_app.flask_config import Config
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import add_item
-from todo_app.data.session_items import get_item
-from todo_app.data.session_items import save_item
+from todo_app.data.session_items import get_items, add_item, get_item, save_item
+from todo_app.data.Items import ToDo
+import requests
+import json
+import os
+import logging
+
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
-def index():
-   
-	return render_template("index.html",items = get_items())
+toDoId = os.environ['TOID']
+doingId = os.environ['DOINGID']
+doneId = os.environ['DONE']
+def getToDo():
+     
+	url = f"https://api.trello.com/1/lists/{toDoId}/cards"
 	
-@app.route('/create', methods =['POST','GET'])
-def create():
-   
-	title = request.form.get('title')
-	added_item	= add_item(title)
+	headers = {
+		"Accept": "application/json"
+	}
+     
+	query = {
+	    'id' : toDoId,
+		'key': os.environ['KEY'],
+		'token' : os.environ['TOKEN']
+		
+	}
+	
+
+	response = requests.request(
+		"GET",
+		url,
+		headers=headers,
+		params=query,
+		
+	)
 	
-	return redirect(url_for('index'))
 	
-@app.route('/save', methods =['POST','GET'])
-def update():
-   #if item is saved then item.status= completed
-    if request.method == 'POST':
-	    get_id = request.form.get('itemId')
+    
+	return response
+
+def getDoing():
```

Comment:
getToDo, getDoing, getDone are exactly the same except for a different list. So you could remove 40 lines of code if you instead of a single function
```
def getItems(idList):
    url = f"https://api.trello.com/1/lists/{idList}/cards"
    etc...
```

Also you could include the conversion from response into a list of your `ToDo` objects in here instead of in `index` so you can treat all three lists consistently. Your current implementation, where you only create the `my_objects` list for to do items, is inconsistent and could cause problems in future